### PR TITLE
Show time elapsed for all tasks and use Duration for the duration and eta.

### LINKED ISSUE #1754: Time elapsed for an operation not shown
The title says it all. The time elapsed when **Loading Chunks** and **Finalizing Octree** is not displayed.
However, the time elapsed when **Rendering** is.

During chunk loading:
image

During octree creation:
image

During rendering:
image

### PR DESCRIPTION
**Pull request by leMaik**
Closes #1754

### CODE REVIEW

File: chunky/src/java/se/llbit/util/ProgressListener.java [hunk 1]
```diff
@@ -17,23 +17,25 @@
...
    */
-  void setProgress(String task, int done, int start, int target);
+  void setProgress(String task, int done, int start, int target, Duration duration);
 
   /**
    * Update progress with ETA.
    */
-  default void setProgress(String task, int done, int start, int target, String eta) {
-    setProgress(task, done, start, target);
+  default void setProgress(String task, int done, int start, int target, Duration duration, Duration eta) {
```
Comment (ShirleyNekoDev) [inline chunky/src/java/se/llbit/util/ProgressListener.java:38]: Same here - is eta "Estimated Time of Arrival" the remaining duration or the total estimated duration of the task?
```suggestion
  default void setProgress(String task, int done, int start, int target, Duration duration, Duration remainingTime) {
```

**Review by ShirleyNekoDev — CHANGES_REQUESTED** [review]
Aside from naming, I really like the use of Duration here :)

File: chunky/src/java/se/llbit/util/ProgressListener.java [hunk 2]
```diff
@@ -17,23 +17,25 @@
...
  */
 public interface ProgressListener {
-  ProgressListener NONE = (task, done, start, target) -> {};
+  ProgressListener NONE = (task, done, start, target, duration) -> {};
 
   /**
    * Update progress without ETA.
    */
-  void setProgress(String task, int done, int start, int target);
+  void setProgress(String task, int done, int start, int target, Duration duration);
```
[the remark this answers is not in the document]
Comment (leMaik) [reply chunky/src/java/se/llbit/util/ProgressListener.java:33]: Good point! I'll rename it to `elapsedTime` which should make it clear without requiring any javadoc.

File: chunky/src/java/se/llbit/util/ProgressListener.java (hunk 1 above)
Comment (leMaik) [reply chunky/src/java/se/llbit/util/ProgressListener.java:38]: I even considered changing this to an `Instant` that has the actual _time_ due to the strict meaning of "ETA" but then decided that we never show the actual time and if we do, we can just calculate it from the remaining time. 

Renaming is a great idea, as above.

**Review by ShirleyNekoDev — APPROVED** [review]

**Pull request merged**